### DISCUSSION TITLE
Error message shows up when trying the test code

### DISCUSSION THREAD

**Discussion by Minghua-UCSD** [blaylockbk/goes2go#13] [Q&A] [answered]
_______________________________
 | Satellite: noaa-goes16      |
 |   Product: ABI-L2-MCMIPC    |
 |    Domain: C                |
Dear Brian, 
I installed this module following the README file and tested the following sample code in my Python:

$ from goes2go.data import goes_latest, goes_nearesttime
$ G_ABI = goes_latest(satellite='G16', product='ABI')

However, the following error message was displayed: 

_**Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/mzheng/goes_data/goes2go/goes2go/data.py", line 542, in goes_latest
    df = _goes_file_df(satellite, product, start, end, refresh=s3_refresh)
  File "/home/mzheng/goes_data/goes2go/goes2go/data.py", line 164, in _goes_file_df
    df.attrs[i] = params[i]
  File "/apps/Anaconda/anaconda3/lib/python3.7/site-packages/pandas/core/generic.py", line 5067, in __getattr__
    return object.__getattribute__(self, name)
AttributeError: 'DataFrame' object has no attribute 'attrs'**_

Do you have any idea what may have caused this problem?

- Minghua Zheng (UC San Diego)

**Comment by blaylockbk** [comment, marked as answer]
Hi Minghua, Thanks for reaching out! 

That error looks like you are using an old version of pandas. Upgrade pandas to the latest version and try again.

Also, I only test goes2go in python 3.8 and 3.9, so I'd recommend upgrading your python.

**Reply by Minghua-UCSD** [reply]
That makes sense to me. I will update my Python version and the pandas.
Thanks, Minghua
